cpp/include/cudf/quantiles.hpp

```diff
@@ -51,33 +30,36 @@ quantile(column_view const& input,
  * four equal-sized groups. See https://en.wikipedia.org/wiki/Quantile
  *
  * The indices used to gather rows are computed by interpolating between the
- * index on either side of the desired quantile. Since some columns may be
- * non-arithmetic, interpolation between rows is limited to non-arithmetic
- * strategies.
+ * index on either side of the desired quantile.
```

Comment:
This documentation isn't correct anymore. You're not interpolating on the index any more, you're interpolating on the index for discrete quantiles and the on the values for arithmetic interpolating schemes.